packages/sync/src/class-functions.php

```diff
@@ -378,6 +378,23 @@ public static function main_network_site_url() {
 		return self::get_protocol_normalized_url( 'main_network_site_url', network_site_url() );
 	}
 
+	/**
+	 * Return main site WordPress.com site ID.
+	 *
+	 * @return string
+	 */
+	public static function main_network_site_wpcom_id() {
+		if ( ! is_multisite() ) {
+			return false;
```

Comment:
Looking at the behavior for main_network_site it utilizes core WP `network_site_url` function which will return the url for both single and multi-site installs. 

We should have consistent functionality and return the  WP.com ID in this function.

Reply:
Updated to return the WP.com ID for single sites, instead of `false`.